nemo_skills/inference/prover.py

```
@@ -0,0 +1,428 @@
+        return code, full_code
+
+    def _transform_for_nemotron_refinement(self, proof_attempt: str, error_message: str) -> list[dict]:
+        """Transform multi-turn refinement into single-turn nemotron-style prompt."""
+        return self.nemotron_refine_prompt.fill(
+            {
+                "proof_attempt": proof_attempt,
+                "error_message": error_message,
+            }
+        )
+
+    async def _single_data_point_generate(self, data_point, data):
```

Ruff (0.14.7)

Unused method argument: `data`

(ARG002)